### DISCUSSION TITLE
Simple components have disappeared from the TailwindCSS website?

### DISCUSSION THREAD

**Discussion by mnapoli** [tailwindlabs/tailwindcss#2557] [Help] [answered]
Hi everyone,

I noticed that the `Components` section on the TailwindCSS website now links to TailwindUI. That makes sense, and I think TailwindUI is awesome.

But now the previous components have disappeared. I know these were limited, and TailwindUI is better, but they were super helpful to get started creating a form, creating a simple button and simple field.

Have these components gone somewhere else?

Thanks for your help!

**Comment by adamwathan** [comment, marked as answer]
Hey @mnapoli! They are still there if you go directly to this page:

https://tailwindcss.com/components

We're a little embarrassed by the quality of them and we were curious about the business impact of linking directly to Tailwind UI instead which is why we're experimenting with that change right now. I think long-term we will revamp that experience from scratch because I totally agree that it's a good idea to provide a set of helpful component examples right in the docs.

**Reply by mnapoli** [reply]
Thanks! They are helpful to avoid the blank page syndrome when getting started (creating a simple button or form _correctly_ always takes a bit of time, so I like to copy-paste those).

I understand that they may not be perfect, but TBH without them I feel kind of lost so I would encourage you not to drop it :) (but that's just my opinion, and I understand that you may have other plans).

Thanks for the answer!
